uint32 state_machine_id
---
string name
bool is_enabled
